pkgs/os-specific/linux/usbtop/default.nix

```diff
@@ -0,0 +1,26 @@
+{ stdenv, fetchFromGitHub
+, cmake
+, libpcap, boost }:
+
+stdenv.mkDerivation rec {
+  pname = "usbtop";
+  version = "1.0";
+
+  src = fetchFromGitHub {
+    owner = "aguinet";
+    repo = pname;
+    rev = "release-${version}";
+    sha256 = "0qbad0aq6j4jrh90l6a0akk71wdzhyzmy6q8wl138axyj2bp9kss";
+  };
+
+  nativeBuildInputs = [ cmake ];
+  buildInputs = [ libpcap boost ];
+
+  meta = with stdenv.lib; {
+    homepage = "https://github.com/aguinet/usbtop";
+    description = "top-like utility that shows an estimated instantaneous bandwidth on USB buses and devices";
```

Comment:
Description should be capitalised

Reply:
Some packages solve this problem by writing "A top-like" or "A 'top'"

Reply:
Fixed